src/logic/documents/scorecards.js

```diff
@@ -368,16 +379,23 @@ const scorecard = ({
     {
       margin: [25, 0, 0, 0],
       table: {
-        widths: ['*', 'auto', 'auto', ...(printStations ? ['auto'] : [])],
+        widths: [
+          '*',
+          getWidthOfBoxOrAuto(t('round'), 25),
```

Comment:
I don't think we need `getWidthOfBoxOrAuto`, when adding a new language, the phrases should be picked such that they fit into the scorecard and we should be fine.

Reply:
It's to prevent this kind of thing, which looks bad IMO:
![skeudenn](https://github.com/user-attachments/assets/7a226a78-bace-4607-85c8-d74fb3a9f1fd)

I think if we have a short word we should be able to use it but for that we need to set a minimum size. If it was the other way around (preventing too long translations) then I would agree with you.

Reply:
Ah, I'm fine with setting a fixed with for all three then, as long as it accommodates the longest event name.